test/test_integration.py

```diff
@@ -51,17 +77,7 @@ def get_script_module(script_name):
 csv_engine.opts = {'engine': 'csv', 'table_name': './{db}_{table}.txt'}
```

Comment:
` csv_engine.opts = {'engine': 'csv', 'table_name': './{db}_{table}.txt'}`

line will be removed